modules/sticky-table-headers/main/index.coffee

```diff
@@ -192,7 +192,7 @@ class StickyTableHeaders
     offsetHeight = 0
     offsetWidth = 0
 
-    if options.fullWidth
+    if options.fullWidth or table.classed('hx-table-full')
```

Comment:
this may be necessary but it is certainly not sufficient, it would make more sense to do this where the options get created.

The main issue is

```
if options.fullWidth
  table.style('min-width', wrapperBox.width + offsetWidth - widthScrollbarOffset - 1 + 'px')
```

Also, I would prefer it if the fullWidth option were completely removed - not only is it undocumented, but you should always be able to work out from context whether to use it or not so it's a bit redundant.